docs/configuration/module/store-and-forward-module.mdx

```diff
@@ -118,7 +118,10 @@ Store and Forward Config options are available for Android.
 <TabItem value="apple">
 
 :::info
-Store and Forward configuration is not currently available via the Apple clients.
+Store and Forward Config options are available for iOS client.
```

Comment:
We'll need to update this to be like the other Modules. They say something like

>All Ambient Lighting Module config options are available on iOS, iPadOS and macOS app versions 2.2.3 and higher at Settings > Modules > Ambient Lighting

@garthvh do you know what version S&F was added to for the Apple apps?

Reply:
2.2.3 and greater is correct, it is somewhat hard to use older App Store versions

Reply:
@moresecode can you please update your text to match the other sections? Thanks!

Reply:
But I'm using version 2.2.23 on my phone.